### DISCUSSION TITLE
Maximum number of UEs per 5G cell

### DISCUSSION THREAD

**Discussion by catalin-nechifor** [srsran/srsRAN_Project#574] [General Help]
Hello,

I have been searching for this info but I can't find anything on it.

How many UEs can be registered simultaneously on a 5G Cell?

I have managed so far to test with 160, and after this I hit this error: 2024-04-24T07:52:21.081146 [DU-MNG  ] [W] ue=160 rnti=0x46b8 proc="UE Create": Procedure failed. Cause: Failed to create DU UE context. Cause: Unable to allocate dedicated PUCCH resources for cell=0.

I have modified some parameters under cell_cfg -> pucch but I still cant pass 160.

Any ideas? 

Thank you!

**Comment by pgawlowicz** [comment]
Could you share your gnb config?
